src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  width: fit-content;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border: 2px solid transparent;
```

Comment:
why you need this border? it is strange

Reply:
If I hadn't added border, there would be stars without right proportions, and tests wouldn't work correctly!

Reply:
you need to fix that by using paddings and margins but not borders